projects/kubernetes-sigs/image-builder/Makefile

```diff
@@ -68,6 +68,7 @@ ADDITIONAL_PAUSE_1-26_FROM=1-26
 ADDITIONAL_PAUSE_1-27_FROM=1-27
 ADDITIONAL_PAUSE_1-28_FROM=1-28
 ADDITIONAL_PAUSE_1-29_FROM=1-28
```

Comment:
is there a reason we're still using 1-28 for 1-29 and 1-30?

Reply:
So the pauseVersion between 1-28, 1-29 and 1-30 didn't change from 3.9 in Kubernetes. This can be verified by checking the [value](https://github.com/kubernetes/kubernetes/blob/v1.30.0-rc.0/cmd/kubeadm/app/constants/constants.go#L436) for different branches 1-28, 1-29 & 1-30. Basically we can use the same value as of 1-28!

Reply:
good explanation!